tests/testthat/test-DiseasystoreBase.R

```diff
@@ -107,52 +149,70 @@ test_that("DiseasystoreBase$determine_new_ranges works", {
 
   conn <- DBI::dbConnect(RSQLite::SQLite())
   ds <- DiseasystoreBase$new(source_conn = "", target_conn = conn, target_schema = target_schema_1)
-  logs <- suppressMessages(SCDB::create_logs_if_missing(paste(target_schema_1, "logs", sep = "."), conn))
-  rows_append(logs, data.frame(date = slice_ts,
-                               table = "table1",
-                               message = glue::glue("ds-range: {start_date} - {end_date}"),
-                               success = TRUE,
-                               log_file = "1"),
-              copy = TRUE, in_place = TRUE)
+  logs <- suppressWarnings(suppressMessages(
+    SCDB::create_logs_if_missing(paste(target_schema_1, "logs", sep = "."), conn)
```

Comment:
Consider being more specific, as SCDB may parse a string to a `DBI::Id` instance:

```suggestion
    SCDB::create_logs_if_missing(DBI::Id(schema = target_schema_1, table = "logs"), conn)
```